### PR TITLE
Fix OSX tests

### PR DESCRIPTION
**Pull request by dzbarsky**
@siddharthab Sorry about missing this in the previous PR, I hadn't bothered checking if we had OSX tests

### CODE REVIEW

**Comment by dzbarsky** [comment]
Ah, so it looks like this works on Bazel 6.x but not 5.1.0. I believe https://github.com/bazelbuild/bazel/commit/33e7261234173fe5f4b6ce12035beeba74d7ae77 added support for the libtool feature in a 6.0pre (May 2022). How do you feel about bumping min bazel to 6.0, or would you rather hold of until 7.0 ships?

**Comment by siddharthab** [comment]
6.0 for minimum supported version is fine. People can always hold off on updating to a new release of this repo, there have not been that many changes.

Will you be able to update the tests and the documentation?

**Comment by dzbarsky** [comment]
Yep just kicked off a run, let's see if it works

**Comment by siddharthab** [comment]
@dzbarsky Are you able to merge these changes yourself? I forgot what the settings on this repo are, and I no longer have access to them.

File: tests/.bazelrc [hunk 1]
```diff
@@ -1 +1,2 @@
 build --incompatible_enable_cc_toolchain_resolution
+build --features=-libtool
```
Comment (keith) [inline tests/.bazelrc:2]: FYI there is `llvm-libtool-darwin` as well that we could use instead

**Comment by dzbarsky** [comment]
Don't think I have permission to merge, I'd need write access.

File: tests/.bazelrc (hunk 1 above)
Comment (dzbarsky) [reply tests/.bazelrc:2]: Oh, interesting. Any pointers to a comparison of the tradeoffs?

Comment (keith) [reply tests/.bazelrc:2]: if this is required for downstream users not requiring it would be the biggest benefit IMO

Comment (siddharthab) [reply tests/.bazelrc:2]: I will merge these changes now to fix the tests. @dzbarsky please consider using llvm-libtool-darwin in a separate change.

**Comment by siddharthab** [comment]
> Don't think I have permission to merge, I'd need write access.

I will see if I can talk to someone with my previous employer to give me admin access again.

**Comment by jsharpe** [comment]
> > Don't think I have permission to merge, I'd need write access.
> 
> I will see if I can talk to someone with my previous employer to give me admin access again.

alternatively we could consider migrating this repo to bazel-contrib? See https://bazel-contrib.github.io/SIG-rules-authors/hosting-policy.html for requirements for doing that.

**Comment by siddharthab** [comment]
> > > Don't think I have permission to merge, I'd need write access.
> > 
> > 
> > I will see if I can talk to someone with my previous employer to give me admin access again.
> 
> alternatively we could consider migrating this repo to bazel-contrib? See https://bazel-contrib.github.io/SIG-rules-authors/hosting-policy.html for requirements for doing that.

Sounds good. There are 3 bazel related repos in the grailbio org, that I don't think anyone actively maintains anymore. I will see if we can get them transferred over to bazel-contrib.

File: tests/.bazelrc (hunk 1 above)
Comment (siddharthab) [reply tests/.bazelrc:2]: I made an attempt in #229 to use llvm-libtool-darwin. It needs more work.

**Pull request merged**